{"name": "test_login_failure_003", "status": "failed", "statusDetails": {"message": "AssertionError: {'ret': 1, 'msg': '用户名/邮箱和密码是必需的。'} != {'ret': 1, 'msg': '无效的 JSON 格式。'}\n- {'msg': '用户名/邮箱和密码是必需的。', 'ret': 1}\n+ {'msg': '无效的 JSON 格式。', 'ret': 1}", "trace": "self = <testplatform.tests.unit.test_Login.TestLogin testMethod=test_login_failure_003>\n\n        def test_login_failure_003(self):\n            response = requests.post('http://127.0.0.1:8000/home/api/login/', json={\n        \"username_or_email\": \"testuser\",\n    })\n            self.assertEqual(response.status_code, 400)\n>           self.assertEqual(response.json(), {\n        \"ret\": 1,\n        \"msg\": \"无效的 JSON 格式。\"\n    })\nE   AssertionError: {'ret': 1, 'msg': '用户名/邮箱和密码是必需的。'} != {'ret': 1, 'msg': '无效的 JSON 格式。'}\nE   - {'msg': '用户名/邮箱和密码是必需的。', 'ret': 1}\nE   + {'msg': '无效的 JSON 格式。', 'ret': 1}\n\ntestplatform\\tests\\unit\\test_Login.py:42: AssertionError"}, "start": 1744076185908, "stop": 1744076185913, "uuid": "28ece01e-4830-47da-bd48-be6b65cd23e0", "historyId": "f0dafca14933db295dd3b618b1678a05", "testCaseId": "f0dafca14933db295dd3b618b1678a05", "fullName": "testplatform.tests.unit.test_Login.TestLogin#test_login_failure_003", "labels": [{"name": "parentSuite", "value": "testplatform.tests.unit"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "HO"}, {"name": "thread", "value": "28800-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testplatform.tests.unit.test_Login"}]}